### DISCUSSION TITLE
Failed to build next js app

### DISCUSSION THREAD

**Discussion by serdar-webdev** [vercel/next.js#67675] [Help] [answered]
### Summary

Hi, i use default next js's configurations but got the errors while building the app;
*here the error logs*
![image](https://github.com/vercel/next.js/assets/115923465/40420c1d-1de9-43a9-a81d-638de1174e39)
*next js config*
![image](https://github.com/vercel/next.js/assets/115923465/383b0b3e-4620-49a2-a1b8-f641480410ee)
*version*
![image](https://github.com/vercel/next.js/assets/115923465/942103de-8bac-4529-8df2-c7d1c1d290a5)
I wondering why i getting these kind of errors, why it cannot read absolute paths, i really need help folks. I will appriciate any help

**Comment by icyJoseph** [comment]
What's your tsconfig.json?

**Reply by serdar-webdev** [reply]
Sorry i'm late, but here the answer. I didn't changed tsconfig.json, i started next js project with default tsconfig.json
![image](https://github.com/user-attachments/assets/b1c959d0-0fd8-4f9e-a376-5fc28bf820f7)

**Reply by icyJoseph** [reply]
And now a screenshot or print out of the applications file tree...

**Reply by serdar-webdev** [reply]
> And now a screenshot or print out of the applications file tree...

is it enough?

![image](https://github.com/user-attachments/assets/2314c6bd-2f3c-4c62-8d1e-f71e4e96b62b)

**Comment by serdar-webdev** [comment, marked as answer]
> ### Summary
> Hi, i use default next js's configurations but got the errors while building the app; _here the error logs_ ![image](https://private-user-images.githubusercontent.com/115923465/347931075-40420c1d-1de9-43a9-a81d-638de1174e39.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0ODEwOTcsIm5iZiI6MTcyMTQ4MDc5NywicGF0aCI6Ii8xMTU5MjM0NjUvMzQ3OTMxMDc1LTQwNDIwYzFkLTFkZTktNDNhOS1hODFkLTYzOGRlMTE3NGUzOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMFQxMzA2MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYWUxMzQ3NmQ2YjdlNDYwMWI5Y2Y1OWRhM2M2NjUzZjhjNTg0MGI0Njg1OTY2NTE3YzA4NjFkZjg0YTZkYTdiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Bd7sU0d0E6c2zGDiPIWGSwEbCDMmi0ex9K9tOOUg1qc) _next js config_ ![image](https://private-user-images.githubusercontent.com/115923465/347931389-383b0b3e-4620-49a2-a1b8-f641480410ee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0ODEwOTcsIm5iZiI6MTcyMTQ4MDc5NywicGF0aCI6Ii8xMTU5MjM0NjUvMzQ3OTMxMzg5LTM4M2IwYjNlLTQ2MjAtNDlhMi1hMWI4LWY2NDE0ODA0MTBlZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMFQxMzA2MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zOTg5M2FjMDNlNzk3MTIzZTI3NDc0ZDE1ZmEzYzBlNjlhY2RkNjhhMWNmMGYwZmI2NjNiZjI4Zjc2NmFmZTk1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.42GcjqxqlFuAW4jjAb5f6H6LwqkNEzuaOgx9YOh23Mg) _version_ ![image](https://private-user-images.githubusercontent.com/115923465/347931524-942103de-8bac-4529-8df2-c7d1c1d290a5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0ODEwOTcsIm5iZiI6MTcyMTQ4MDc5NywicGF0aCI6Ii8xMTU5MjM0NjUvMzQ3OTMxNTI0LTk0MjEwM2RlLThiYWMtNDUyOS04ZGYyLWM3ZDFjMWQyOTBhNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMFQxMzA2MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NGVjMDRlMTk1ZTQ5ZDllZTNlNmFkM2E4NTk3YjU2YjczOGI2N2E5ZjQ5NDQ5Mzg1MDMyNWU2ZWNiMDYzNWUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.PyVybXzJErVv3m3HMnSCn37vwqUA9FmI03dDgxsLBZA) I wondering why i getting these kind of errors, why it cannot read absolute paths, i really need help folks. I will appriciate any help
> 
> ### Additional information
> _No response_
> 
> ### Example
> _No response_

just never mind, we fixed it just redeploying the app.
